externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/events/EventLoggingServiceSuite.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.kyuubi.operation.{JDBCTestUtils, OperationHandle}
 
 class EventLoggingServiceSuite extends WithSparkSQLEngine with JDBCTestUtils {
 
-  private val logRoot = Utils.createTempDir()
+  private val logRoot = "file:" + Utils.createTempDir()
```

Comment:
`logRoot` from `Path` type to `String` type, so we don't need to call `toString` again.

```
  private val logRoot = "file:" + Utils.createTempDir()
  private val currentDate = Utils.getDateFromTimestamp(System.currentTimeMillis())


  override def withKyuubiConf: Map[String, String] = Map(
    KyuubiConf.ENGINE_EVENT_LOGGERS.key -> s"$JSON,$SPARK",
    KyuubiConf.ENGINE_EVENT_JSON_LOG_PATH.key -> logRoot.toString,
    "spark.eventLog.enabled" -> "true",
    "spark.eventLog.dir" -> logRoot.toString
  )
```

Reply:
it's good.
i don't find this problem.